task/upload.go

```diff
@@ -511,8 +511,12 @@ func removeCredentials(metadata *clients.CatalystCallback) *clients.CatalystCall
 
 func uploadTaskOutputLocations(tctx *TaskContext) ([]OutputName, []clients.OutputLocation, error) {
 	playbackId := tctx.OutputAsset.PlaybackID
+	generateMp4s := false
+	if tctx.OutputAsset.GenerateMp4s {
+		generateMp4s = true
+	}
```

Comment:
any reason for the indirection?
```suggestion
	generateMp4s := tctx.OutputAsset.GenerateMp4s
```